src/pages/basic-concepts/content-markdown.md

```diff
@@ -3,7 +3,143 @@ layout: ~/layouts/Main.astro
 title: Content (Markdown)
 ---
 
-- Layouts
-- MDX
+Astro comes with out-of-the-box Markdown support powered by the expansive [remark](https://remark.js.org/) ecosystem.
 
-## Remote data
+### Remark Plugins
+
+**This is the first draft of Markdown support!** While we plan to support user-provided `remark` plugins soon, our hope is that you won't need `remark` plugins at all!
+
+In addition to [custom components inside the `<Markdown>` component](https://github.com/snowpackjs/astro/blob/main/docs/markdown.md#markdown-component), Astro comes with [Github-flavored Markdown](https://github.github.com/gfm/) support, [Footnotes](https://github.com/remarkjs/remark-footnotes) syntax, [Smartypants](https://github.com/silvenon/remark-smartypants), and syntax highlighting via [Prism](https://prismjs.com/) pre-enabled. These features are likely to be configurable in the future.
+
+### Markdown Pages
+
+Astro treats any `.md` files inside of the `/src/pages` directory as pages. These pages are processed as plain Markdown files and do not support components. If you're looking to embed rich components in your Markdown, take a look at the [Markdown Component](#markdown-component) section.
+
+`layout`
+
+The only special Frontmatter key is `layout`, which defines the relative path to a `.astro` component which should wrap your Markdown content.
+
+`src/pages/index.md`
+
+```jsx
+---
+layout: ../layouts/main.astro
+---
+
+# Hello World!
+```
+
+Layout files are normal `.astro` components. Any Frontmatter defined in your `.md` page will be exposed to the Layout component as the `content` prop. `content` also has an `astro` key which holds special metadata about your file, like the complete Markdown `source` and a `headings` object.
+
+The rendered Markdown content is placed into the default `<Slot />` element.
+
+`src/layouts/main.astro`
+
+```jsx
+---
+export let content;
+---
+
+<html>
+  <head>
+    <title>{content.title}</title>
+  </head>
+
+  <body>
+    <slot/>
+  </body>
+</html>
+```
+
+### Markdown Component
+
+Similar to tools like [MDX](https://mdxjs.com/) or [MDsveX](https://mdsvex.com/), Astro makes it straightforward to embed rich, interactive components inside your Markdown content. The `<Markdown>` component is statically rendered, so it does not add any runtime overhead.
```

Comment:
I think this would read better as:

### Astro's Markdown Component

<!-- If we are importing something make it clear at the top that we are importing an astro component, this detail can get lost in the code block if your not trained to notice it -->
Astro has a dedicated component used to let you render your markdown as HTML components. This is a special component that is only exposed on `.astro` files.  To use `<Markdown>` component, within your frontmatter block use the following import statement.
```js
---
import { Markdown } from 'astro/components';
---
```
To utilise this within your `.astro` file you can do so in the following way:

```js
---
import { Markdown } from 'astro/components';
---

<Layout>
  <Markdown>
    # Hello world!
     The contents inside here is all in markdown. 
  </Markdown>
</Layout>
```

`<Markdown>` components provide more flexibility and allows you to use plain HTML, or any other type of [components that is supported](#) by Astro. <!-- link to supported UI frameworks and components list-->As seen in our next example:

```js
---
import { Markdown } from 'astro/components';
import Layout from '../layouts/main.astro';
import MyFancyCodePreview from '../components/MyFancyCodePreview.tsx';
const expressions = 'Lorem ipsum';
---
<Layout>
  <Markdown>
    # Hello world!
    **Everything** supported in a `.md` file is also supported here!
    There is _zero_ runtime overhead.
    In addition, Astro supports:
    - Astro {expressions}
    - Automatic indentation normalization
    - Automatic escaping of expressions inside code blocks
    ```jsx
      // This content is not transformed!
      const object = { someOtherValue };
    ```

    - Rich component support like any `.astro` file!
    - Recursive Markdown support (Component children are also processed as Markdown)

    <MyFancyCodePreview:visible>
      ```jsx
      const object = { someOtherValue };
      ```
    </MyFancyCodePreview:visible>
  </Markdown>
</Layout>
```